include/swift/AST/AvailabilitySpec.h

```diff
@@ -101,38 +104,50 @@ class PlatformVersionConstraintAvailabilitySpec : public AvailabilitySpec {
 };
 
 /// An availability specification that guards execution based on the
-/// compile-time language version, e.g., swift >= 3.0.1.
-class LanguageVersionConstraintAvailabilitySpec : public AvailabilitySpec {
-  SourceLoc SwiftLoc;
+/// compile-time platform agnostic version, e.g., swift >= 3.0.1,
+/// package-description >= 4.0.
+class PlatformAgnosticVersionConstraintAvailabilitySpec : public AvailabilitySpec {
+  SourceLoc PlatformAgnosticNameLoc;
 
   llvm::VersionTuple Version;
   SourceRange VersionSrcRange;
 
 public:
-  LanguageVersionConstraintAvailabilitySpec(SourceLoc SwiftLoc,
-                                            llvm::VersionTuple Version,
-                                            SourceRange VersionSrcRange)
-    : AvailabilitySpec(AvailabilitySpecKind::LanguageVersionConstraint),
-      SwiftLoc(SwiftLoc), Version(Version),
-      VersionSrcRange(VersionSrcRange) {}
+  PlatformAgnosticVersionConstraintAvailabilitySpec(AvailabilitySpecKind
+                                                      AvailabilitySpecKind,
+                                                    SourceLoc
+                                                      PlatformAgnosticNameLoc,
+                                                    llvm::VersionTuple Version,
+                                                    SourceRange VersionSrcRange)
```

Comment:
Nitpick: Please don't split names from types like this. Just wrap after the open-brace and double-indent.

(You can use clang-format to stay consistent with our style, but I don't remember how easy that is to set up.)

Reply:
oh wow, clang-format is awesome! I wish you would have told me about clang-format before 😂